include/oneapi/dpl/internal/function.h

```diff
@@ -63,12 +64,25 @@ using oneapi::dpl::__par_backend_hetero::__internal::__buffer;
 // algorithm from either a SYCL iterator or a USM pointer.
 template <sycl::access::mode Mode, typename Policy, typename Iterator>
 auto
-get_access(Policy, Iterator i, typename ::std::enable_if<is_hetero_iterator<Iterator>::value, void>::type* = nullptr)
```

Comment:
Is there code which uses that function?
I guess that function is not used any more or probably I'm wrong?

Reply:
You are right but I hadn't goal to remove any not used functions in this PR.